testcases/OpTestLPM.py

```diff
@@ -104,6 +162,7 @@ def tearDown(self):
         if self.firewall_status:
             self.cv_HOST.host_run_command("systemctl start firewalld.service")
 
+
 def LPM_suite():
     s = unittest.TestSuite()
     s.addTest(OpTestLPM())
```

Comment:
As we are running this as test suite, can we have as following
1. Basic LPM
2. LPM with vnic cards

So that if any one wants to run basic test, can run it.